ajax/libs/Bounce/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "bounce",
+  "filename": "bounce.min.js"
+  "version": "0.0.0",
```

Comment:
Let's remove this version field.
Because you sent this PR via package.json.
You could refer #12464's `Via single package.json`, this field is not a `Necessary` or `Unnecessary` one.
Many thanks.